src/clj_zeppelin/core.clj

```diff
@@ -77,7 +82,8 @@
   [notebook-api-url note_json_url]
   (let [resp @(ht/request {:url (str notebook-api-url "import")
                            :method :post
-                           :body (finalize (json/read-str (slurp note_json_url):key-fn keyword))})]
+                           :body (json-convert note_json_url )
```

Comment:
There can be 2 different types of errors:

- the note_json_url cannot be read
- the notebook-api-url throws an error on import. 

These should report 2 different errors,
Right now note_json_url's error is silently swallowed.